src/migration_state.rs

```diff
@@ -95,7 +95,7 @@ impl MigrationState {
             })
             .while_some()
             .collect::<Vec<_>>();
-        let next_id = migrations.iter().map(|m| m.id).last().unwrap_or(0) + 1;
+        let next_id = migrations.iter().map(|m| m.id).next_back().unwrap_or(0) + 1;
```

Comment:
IMO, `last` is more meaningful to me over [`next_back`](https://doc.rust-lang.org/std/array/struct.IntoIter.html#method.next_back). The clippy folks are legit and I trust them. Interesting that there's not a lot new dialog on https://github.com/rust-lang/rust-clippy/issues/1822.